medusa/providers/torrent/json/btn.py

```diff
@@ -269,13 +266,15 @@ def _episode_search_params(ep_obj):
         return to_return
 
     def _api_call(self, apikey, params=None, results_per_page=300, offset=0):
-
+        """Call provider API."""
         server = jsonrpclib.Server(self.urls['base_url'])
         parsed_json = {}
 
         try:
             parsed_json = server.getTorrents(apikey, params or {}, int(results_per_page), int(offset))
             time.sleep(cpu_presets[app.CPU_PRESET])
+            if parsed_json.get('results') == '0':
+                return {}
 
         except jsonrpclib.jsonrpc.ProtocolError as error:
             if error.message == 'Call Limit Exceeded':
```

Comment:
Should check status code `503 Service Unavailable: invalid API Key, or you have hit the API too much`